src/libraries/System.Runtime/ref/System.Runtime.cs

```diff
@@ -10508,6 +10508,7 @@ public sealed partial class SecurityElement
         public string? Attribute(string name) { throw null; }
         public System.Security.SecurityElement Copy() { throw null; }
         public bool Equal([System.Diagnostics.CodeAnalysis.NotNullWhenAttribute(true)] System.Security.SecurityElement? other) { throw null; }
+        [return: NotNullIfNotNull("str")]
```

Comment:
In ref assemblies, attributes must be fully qualified: `[return: System.Diagnostics.BlaBlah...()]`